python_src/src/lamp_py/postgres/seed_metadata.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import sys
+from typing import List
+
+import sqlalchemy as sa
+
+from lamp_py.runtime_utils.import_env import load_environment
+
+from .postgres_schema import (
+    MetadataLog,
+    SqlBase,
+    VehicleEventMetrics,
+    VehicleEvents,
+    VehicleTrips,
+)
+from .postgres_utils import DatabaseManager
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Interact with Performance Manager RDS for Testing"""
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+
+    parser.add_argument(
+        "--env-file",
+        dest="env_file",
+        help="environment file containing rds connection parameters",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        dest="verbose",
+        help="if set, use debug logging",
+    )
+
+    parser.add_argument(
+        "--seed-file",
+        dest="seed_file",
+        help="if set, read a json file to seed the metadata table with",
+    )
+
+    parser.add_argument(
+        "--clear-rt",
+        action="store_true",
+        dest="clear_rt",
+        help="if set, clear gtfs-rt database tables",
+    )
+
+    parser.add_argument(
+        "--clear-static",
+        action="store_true",
+        dest="clear_static",
+        help="if set, clear gtfs static database tables and rt tables",
+    )
+
+    return parser.parse_args(args)
+
+
+def seed_metadata(db_manager: DatabaseManager, seed_file: str) -> None:
+    """
+    seed metadata table for dev environment
+    """
+    logging.info("Seeding Metadata From File %s", seed_file)
+
+    try:
+        with open(seed_file, "r", encoding="utf8") as seed_json:
+            paths = json.load(seed_json)
+
+        db_manager.add_metadata_paths(paths)
+
```

Comment:
Probably makes sense to move this `add_metadata_paths` function out of `DatabaseManager` if this is the only part of the application using that functionality, and it's not really generic to DB interaction.

Reply:
i left this in because some tests were using this as well, although i agree it would be a good idea to get rid of it from this file eventually.